### ISSUE TITLE
News reader and events reader with a message or default page if item-url empty

### ISSUE THREAD

**Issue by havutcuoglu** [contao/core#8769]
Hi contao team,

i have a news page with news (**domain.com/news.html**) and a news reader modul in a detail page (**domain.com/news/news-detail.html**) in Contao 3.5.28.
For example, if i click a news item it's open in the detail page (**domain.com/news/news-detail/news-number-one.hmtl**). But if i call the url direct (**domain.com/news/news-detail.html**) i get a clean detail page. This page will also registered from search engines.

Can we add in the news reader modul a checkbox for a default page or a message like this? "You call the page direktly. Please go to news page <link> and choose a news item" .

Thanks alot

**Comment by asaage** [comment]
Often discussed but no general solution yet...
related: https://github.com/contao/core/issues/8361
or https://github.com/contao/core/issues/8367

In case you prefer a message: you should be able to do that in the news-reader-template.

**Comment by havutcuoglu** [comment]
Yes, but this is not only for me. I think, is better send a 404 header for the empty page via checkbox in modul. The message is not important, important thing is the SEO. You have a empty page in the search results from search engines.

**Comment by asaage** [comment]
>important thing is the SEO

agree - 
that's why i think it is more important to get it out of the sitemap.xml ;)

**Comment by havutcuoglu** [comment]
That's right. 👍
